orderer/common/multichannel/metrics.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package multichannel
+
+import (
+	"github.com/hyperledger/fabric/common/metrics"
+	"github.com/hyperledger/fabric/orderer/common/types"
+)
+
+var (
+	clusterStatusOpts = metrics.GaugeOpts{
+		Namespace:    "cluster",
```

Comment:
So, we already have a "cluster" namespace, used by `orderer/common/cluster`, but it does specify a subsystem of `comm`.  Perhaps we should add a subsystem of `participation` here? Or pick a different namespace?

Reply:
I definitely like the idea of adding "participation" as the subsystem. And I wasn't in love with using "cluster" as the namespace. Maybe it should really be "channel"?

Reply:
Went with "cluster" and "participation" for now (and updated the help text).

Reply:
The more I think on this, the more convinced I am that 'cluster' isn't right, and I'm a little confused why we call it cluster status at all.  It has nothing to do with the cluster? Being a follower does not make you a member of the cluster... why not a vanilla channel_participation?

Reply:
So are you suggesting we change the clusterRelation in general reported by the channel participation API in addition to the metric? We could rename that to Relation if it's not cluster-related and then the metric could be namepsace "channel" subsystem "participation" and name "relation" for channel_participation_relation.

Reply:
Updated to what I mentioned in my previous comment.

Reply:
Updated to "participation" for the namespace and no subsystem.